tests/test_dagfactory.py

```diff
@@ -429,7 +429,15 @@ def test_load_yaml_dags_succeed(mock_emit_usage_metrics_if_enabled):
         dags_folder="tests/fixtures",
         suffix=["dag_factory_variables_as_arguments.yml"],
     )
+
     # Confirm the representative telemetry for all the DAGs defined in the desired YAML is being sent
     args = mock_emit_usage_metrics_if_enabled.call_args.args
     assert args[0] == "load_yaml_dags"
     assert args[1] == {"dags_count": 2, "tasks_count": 4, "taskgroups_count": 0}
+    
+    # test load with none suffix
+    load_yaml_dags(
+        globals_dict=globals(),
+        dags_folder="tests/fixtures",
+    )
+    
```

Comment:
It would be great to have a dedicated test for this use case, having asserts that confirm the behaviour is as expected.

Reply:
Moved this into a new test and added assert to check the log